app/cdap/components/ConnectionList/__test__/ConnectionList.test.tsx

```diff
@@ -14,29 +14,24 @@
  * the License.
  */
 
-import { act, fireEvent, getByTestId, render, screen } from '@testing-library/react';
+import { render, screen } from '@testing-library/react';
+import * as apiHelpersForExploreConnection from 'components/Connections/Browser/GenericBrowser/apiHelpers';
 import * as apiHelpers from 'components/Connections/Browser/SidePanel/apiHelpers';
 import * as reducer from 'components/Connections/Create/reducer';
-import { createBrowserHistory as createHistory } from 'history';
 import React from 'react';
 import { Route, Router, Switch } from 'react-router';
+import history from 'services/history';
 import ConnectionList from '..';
```

Comment:
please provide full path

Reply:
This line doesn't come under updated code